common/include/pcl/register_point_struct.h

```diff
@@ -189,6 +189,23 @@ namespace pcl
       for (std::uint32_t i = 0; i < count; ++i)
         p[i] /= scalar;
     }
+
+    template<typename T1, typename T2> inline
```

Comment:
Thoughts on calling it ArrayT and ScalarT?

Reply:
Renaming T2 to ScalarT would fit, but T1 is not an array type in both versions of `divscalar2` (in this version `std::is_array<T1>` returned false). I could rename T1 to `NoArrayT` here and to `ArrayT` below, if you think that makes it clearer?

Reply:
Yes please